src/cf-typography/usage.md

```diff
@@ -427,103 +331,94 @@ This is just another set of variations to show off what you could do.
 
 <p>
     For more information, email
-    <a class="a-icon-link a-icon-link__mail a-icon-link__no-wrap" href="#">
-        <span class="a-icon-link_text">john.smith@cfpb.gov</span>
+    <a class="a-link a-link__icon a-link__mail a-link__no-wrap" href="#">
+        <span class="a-link_text">john.smith@cfpb.gov</span>
     </a>.
 </p>
 
 ```
 <p>
     For more information, email
-    <a class="a-icon-link a-icon-link__mail a-icon-link__no-wrap" href="#">
-        <span class="a-icon-link_text">john.smith@cfpb.gov</span>
+    <a class="a-link a-link__icon a-link__mail a-link__no-wrap" href="#">
+        <span class="a-link_text">john.smith@cfpb.gov</span>
     </a>.
 </p>
 ```
 
-
-### Styled link
-
-#### Default pattern
-
-<a class="a-styled-link" href="#">
-    Default styled link
-</a>
-
-```
-<a class="a-styled-link" href="#">
-    Default styled link
-</a>
-```
-
 ### Jump link
 
-#### Default pattern
+### Jump link
```

Comment:
Should remain h4; would not have it be the exact same wording as its parent heading.

Or... perhaps you meant to only have one of those.